travis/rspec_integration_tests

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+from test_server import main as test_server_main
+import time
+import subprocess
+import threading
+import os
+
+
+class TestServerThread(threading.Thread):
+    def run(self):
+        test_server_main()
+
+def main(argv=None):
+    """Run integration tests with RSpec"""
```

Comment:
unnecessary comment

Reply:
it is the description of the method, it is encouraged https://www.python.org/dev/peps/pep-0257/

Reply:
👍